geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/planning/LocalQueryRunner.scala

```diff
@@ -95,7 +104,7 @@ abstract class InMemoryQueryRunner(stats: GeoMesaStats, authProvider: Option[Aut
   }
 }
 
-object InMemoryQueryRunner {
+object LocalQueryRunner {
```

Comment:
Did you combine the foo for the 'InMemory' CQ QueryRunner with the stuff necessary for C* and BigTable?

Reply:
Yeah, this should be an easy way to get all our query handling/filtering/transforming against any random iterator of features. And since it's all in one place, we save a lot of redundancy.